google/cloud/spanner_v1/_opentelemetry_tracing.py

```diff
@@ -78,8 +78,17 @@ def trace_call(name, session, extra_attributes=None):
         try:
             yield span
         except Exception as error:
-            span.set_status(Status(StatusCode.ERROR, str(error)))
-            span.record_exception(error)
+            set_span_error_and_record_exception(span, error)
             raise
         else:
             span.set_status(Status(StatusCode.OK))
+
+
+def set_span_error_and_record_exception(span, exc):
```

Comment:
Since this is not used anywhere else, we can revert it back as it was.

Reply:
I did use it in the except block @harshachinta